victim-identifier/README.md

```diff
@@ -0,0 +1,66 @@
+# Victim Identifier Bot
+
+## Description
+
+This bot identifies possible victims during the preparation or the exploitation stage of an attack, using the `VictimIdentifier` library from the [General-Agents-Module](https://github.com/NethermindEth/general-agents-module#victimidentifier)
+
+> The bot uses Luabase, Moralis, Ethplorer and block explorer APIs. In order to increase performance, add your own API keys to the `src/keys.ts` file.
+
+## Supported Chains
+
+- Ethereum
+- Optimism
+- BNB Smart Chain
+- Polygon
+- Fantom
+- Arbitrum
+- Avalanche
+
+## Alerts
+
+- VICTIM-IDENTIFIER-PREPARATION-STAGE
+
+  - Fired when an contract address is identified as a possible victim in the preparation stage of an attack
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata contains a record of contract address to an object with the following properties:
+    - `protocolUrl`: The victim protocol's website address, if found.
+    - `protocolTwitter`: The victim protocol's twitter handle, if found.
+    - `tag`: The victim protocol's name, if found.
+    - `holders`: The token holders of the victim if it is an ERC20 token.
```

Comment:
Should add a caveat since this property, under certain conditions, will not have a value (when there are a high number of victims)